mlflow/server/__init__.py

```diff
@@ -148,3 +154,7 @@ def _run_server(
     else:
         full_command = _build_gunicorn_command(gunicorn_opts, host, port, workers or 4)
     exec_cmd(full_command, env=env_map, stream_output=True)
+
+
+application = app
+application = make_app_with_telemetry(application)
```

Comment:
the gunicorn command looks for `mlflow.server:app` so i think this variable would also need to be named `app` rather than `application`

Reply:
Done